core/commands/generate_repository_partials/mdbe_ci_parser.rb

```diff
@@ -38,6 +38,7 @@ def self.parse_mdbe_ci_rpm_repository(config, product_version, auth, log, logger
       append_url(%w[yum]),
       split_rpm_platforms,
       extract_field(:platform_version, %r{^(\p{Digit}+)\/?$}),
+      append_url(%w[x86_64], nil, false, true),
```

Comment:
This is not correct use of `append_url`. If you want unconditional addition, then create a new prefix.